form_submit/views.py

```diff
@@ -4,10 +4,15 @@
 import subprocess
 import os
 from pdflatex import PDFLaTeX
+from aide_validation.validator import Validator
 
 LANGUAGES = [("es", "Español"), ("en", "English")]
```

Comment:
Could we remove the Spanish option from here for now? #6

Reply:
I went back and forth on this because the Design Specs are available in both Spanish and English but the validation report is available in English only. I wasn't sure whether to have an invalid option (Validation Report - Spanish) or mask a valid one (Design Specs - Spanish), and in the end I went with the second one since AIDE is still in development and the website isn't really live

Reply:
I'm very open to changing that though. The ideal would be if the `language` dropdown only displayed after a report type was selected, that way only supported languages could be listed. The issue is I don't know how to do that, but maybe it could be a student project?

A short term fix is to remove the `language` option and just include everything within the same dropdown (like I do for HTML/PDF on the design specs.). I.e. the dropdown would be: `Design Specs - PDF - Spanish`, `Design Specs - HTML - Spanish`, `Design Specs - PDF - English`, `Design Specs - HTML - English`, `Validation Report - PDF - English`. My worry is that will get quite long, but could be the best fix while we figure out how to implement languages like I described in the first paragraph?

Reply:
Oh I see, I didn't notice that complication. I think the short term fix you mentioned is fine for now, and maybe we could put the language in parentheses to help with readability and separate it from the other descriptors. The ideal fix you mentioned could probably be implemented in `class DocGenForm`, and we could just leave it as a future goal.

Looks good to merge!